react-events-and-state/package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "react-events-and-state",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "build": "webpack",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC"
+}
```

Comment:
Make sure you are installing your dependencies and devDependencies for each exercise